src/azmi-commandline/azmi-commandline.cs

```diff
@@ -5,10 +5,21 @@
 using System.CommandLine.Invocation;
 
 namespace azmi_commandline
-{
-
+{    
     class Program
     {
+        class getBlobsOptionsType
```

Comment:
Please, add some TODO: comment here. Unless cleaned up, this will bite us in future.